nx-dev/ui-customers/src/lib/oss-projects.tsx

```diff
@@ -0,0 +1,224 @@
+
+const variants = {
+  hidden: {
+    opacity: 0,
+    transition: {
+      when: 'afterChildren',
+    },
+  },
+  visible: (i: number) => ({
+    opacity: 1,
+    transition: {
+      delay: i || 0,
+    },
+  }),
+};
+const itemVariants = {
+  visible: (i: number) => ({
+    opacity: 1,
+    y: 0,
+    transition: {
+      delay: i * 0.25,
+      duration: 0.65,
+      ease: 'easeOut',
+      when: 'beforeChildren',
+      staggerChildren: 0.3,
+    },
+  }),
+  hidden: {
+    opacity: 0,
+    y: 4,
+    transition: {
+      when: 'afterChildren',
+    },
+  },
+};
+
+const ossProjects = [
+  {
+    name: 'Tanstack',
+    logo: TanstackIcon,
+    url: 'https://github.com/tanstack',
+  },
+  {
+    name: 'EpicWeb',
+    logo: EpicWebIcon,
+    url: 'https://github.com/epicweb-dev/epicshop',
+  },
+  {
+    name: 'RedwoodJS',
+    logo: RedwoodJsIcon,
+    url: 'https://redwoodjs.com/',
+  },
+  {
+    name: 'Storybook',
+    logo: StorybookIcon,
+    url: 'https://github.com/storybookjs/storybook',
+  },
+  {
+    name: 'Cypress',
+    logo: null,
+    url: 'https://github.com/cypress-io/cypress',
+  },
+  {
+    name: 'Lerna',
+    logo: LernaIcon,
+    url: 'https://lerna.js.org',
+  },
+  {
+    name: 'BuilderIO Mitosis',
+    logo: null,
+    url: 'https://github.com/BuilderIO/mitosis',
+  },
+  {
+    name: 'RxJS',
+    logo: RxJSIcon,
+    url: 'https://github.com/ReactiveX/rxjs',
+  },
+  {
+    name: 'Electron Forge',
+    logo: null,
+    url: 'https://github.com/electron/forge',
+  },
+  {
+    name: 'TypeScript ESLint',
+    logo: null,
+    url: 'https://github.com/typescript-eslint/typescript-eslint',
+  },
+  {
+    name: 'Angular ESLint',
+    logo: null,
+    url: 'https://github.com/angular-eslint/angular-eslint',
+  },
+  {
+    name: 'Strapi',
+    logo: null,
+    url: 'https://github.com/strapi/strapi',
+  },
+  {
+    name: 'Sentry',
+    logo: SentryIcon,
+    url: 'https://github.com/getsentry/sentry-javascript',
+  },
+  {
+    name: 'MUI',
+    logo: MuiIcon,
+    url: 'https://github.com/mui/material-ui',
+  },
+];
+export function OssProjects(): JSX.Element {
+  return (
+    <section>
+      <div className="mx-auto max-w-7xl text-center">
+        <SectionHeading as="h2" variant="subtitle">
+          Popular OSS projects using Nx
+        </SectionHeading>
+
+        <div className="mt-20">
+          <motion.dl
+            initial="hidden"
+            variants={variants}
+            whileInView="visible"
+            viewport={{ once: true }}
+            className="grid grid-cols-2 justify-between sm:grid-cols-4"
+          >
+            <motion.div
+              custom={1}
+              variants={itemVariants}
+              className="flex items-center justify-center border border-slate-200/20 p-12 transition hover:bg-slate-100/25 dark:border-slate-800/20 dark:hover:border-slate-600/20 dark:hover:bg-slate-600/10"
+            >
+              <Link href="https://github.com/tanstack'" target="_blank">
```

Comment:
`Link` component is primarily used to navigate between Nextjs routes, and pre-fetch the content. Not need in this context, prefer use a standard `a` tag.